source/reference/known-issues.txt

```diff
@@ -0,0 +1,56 @@
+=====================
+Known Issues for |bi|
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+SQL Compatibility Issues
+------------------------
+
+|bi-short| Returns Empty Set Where MySQL Returns ``NULL`` Values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some `SQL aggregate functions <https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html>`__
+(like ``SUM``, ``MIN``, or ``COUNT``) return ``NULL`` if there are no
+matching values for that aggregate function. This response is
+effectively an empty row.
+
+Given the exact same SQL query, the |bi-short| does not return
+``NULL`` for any aggregate functions; it returns an empty set instead.
+
+.. example::
+
+   Using Tableau, you query a MySQL database of students to find the
+   sum, count, highest, and lowest test scores by each class.
+
+   Unknown to you, this is a new database table created for the new
+   classes of students, so there is no data in this table yet. This SQL
+   query would return no students because no scores had been entered:
+
+   .. code-block:: sql
+
+      SELECT SUM(scores), COUNT(*), COUNT(scores), MAX(scores), MIN(scores) FROM students_2019 GROUP BY class;
+
+      +-------------+----------+---------------+-------------+-------------+
+      | sum(scores) | count(*) | count(scores) | max(scores) | min(scores) |
+      +-------------+----------+---------------+-------------+-------------+
+      |   NULL      |        0 |             0 |   NULL      |   NULL      |
+      +-------------+----------+---------------+-------------+-------------+
+ 
+   If you query the |bi-short| for this same data, it returns an 
+   empty data set. MongoDB returns nothing if no data exists for a 
+   value, its fields is not part of the document. Therefore, it cannot 
```

Comment:
This sentence doesn't really make sense.
`MongoDB returns nothing if no data exists for a value, its fields is not part of the document.`

Reply:
N/A. Fixed.